packages/web/src/provider/provider.ts

```diff
@@ -71,4 +71,12 @@ export interface Provider extends CommonProvider<ClientProviderStatus> {
     context: EvaluationContext,
     logger: Logger,
   ): ResolutionDetails<T>;
+
+  /**
+   * Track a thing
+   * @param occurrenceKey
+   * @param context
+   * @param occurrenceDetails
+   */
+    track?(occurrenceKey: string, context: EvaluationContext, occurrenceDetails: OccurrenceDetails): void;
```

Comment:
I think `event` or `eventName` is more idiomatic and recognizable than `occurenceKey` (closer to the defacto standard of [Segment's SDK](https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/#track)) 
```suggestion
    track?(event: string, context: EvaluationContext, occurrenceDetails: OccurrenceDetails): void;
```